### DISCUSSION TITLE
QQ版本过低

### DISCUSSION THREAD

**Discussion by 7FreezingPoint** [simple-robot/Discussions#141] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

simboot-core-spring-boot-starter版本：3.1.0
simbot-component-mirai-core版本：3.0.0.0-RC.2
code=235, title=温馨提示, message=当前QQ版本过低，请升级至最新版本后再登录。点击进入下载页面
7号还在正常运行，8号就不行了

**Comment by ForteScarlet** [comment]
参考 https://github.com/orgs/simple-robot/discussions/135 
这是mirai目前的通病
